### PR TITLE
Add support for multiple brands

### PR DESCRIPTION
**Pull request by SerhiiBovtriuk**

### CODE REVIEW

**Comment by jorunfa** [comment]
Is the button expected to stay the same, or is it supposed to have a different look for different brands?
Could either add all of that (if that's the plan), or rename the pull request (add support for multiple brand endpoints)

**Comment by SerhiiBovtriuk** [comment]
I didn't get clear message from anybody about the button status for now, so this question is still in the air. I will update this when there will be more details.

File: connect/src/com/telenor/connect/id/IdProvider.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.telenor.connect.id;
+
+public enum IdProvider {
+    CONNECT_ID("connect.telenordigital.com", "connect.staging.telenordigital.com"),
+    TELENOR_ID("example.com", "staging-telenorid.com"); // TODO: production is not set yet
```
Comment (vlarsen) [inline connect/src/com/telenor/connect/id/IdProvider.java]: I think the decision is to not use toplevel domain for the IDP, but rather signin.staging-telenorid.com

Comment (SylviaBrudeli) [reply connect/src/com/telenor/connect/id/IdProvider.java]: Yes, you are right Vidar!

**Pull request merged**